test/index.js

```diff
@@ -43,6 +43,22 @@ describe('init', () => {
     FullStory.init({ orgId: testOrg });
     expect(window._fs_org).to.equal(testOrg);
   });
+
+  it('should add _fs_run_in_iframe value to window object', () => {
+    FullStory.init({
+      orgId: testOrg,
+      iFrameOption: FullStory.IFrameOption.RecordCrossDomainIFrame,
```

Comment:
Per my other comment: I think you're fine to use string values in JavaScript. No need to create a dictionary in JavaScript.

Reply:
Similar to my comment above, this favors using constants defined by the SDK rather than strings.  If we use the string literal, then we lose testability of whether `FullStory.IFrameOption.RecordCrossDomainIFrame` actually set the correct window param.  Cause all the index code does is take the value from `iFrameOption` and assign it to `window[iFrameOption]`.